pi-manage

```diff
@@ -584,16 +585,24 @@ def createdb(stamp=False):
 @manager.command
 def dropdb(dropit=None):
     """
-    This drops all the privacyIDEA database tables (except audit table).
+    This drops all the privacyIDEA database tables.
     Use with caution! All data will be lost!
 
     For safety reason you need to pass
         --dropit==yes
     Otherwise the command will not drop anything.
     """
     if dropit == "yes":
+        engine = db.session.get_bind()
```

Comment:
We can rewrite this as
```python
    if dropit == "yes":
        print("Dropping all database tables!")
        db.drop_all()
        table_name = "alembic_version"
        db.reflect()
        table = db.Model.metadata.tables.get(table_name, None)
        if table:
            db.Model.metadata.drop_all(bind=db.engine,
                                       tables=[table],
                                       checkfirst=True)
    else:
        print("Not dropping anything!")
```
This way we reduce imports and use the given database object

Reply:
fixed in commit 7dd23f.

Reply:
The formatting seems reasonable so i'll merge all of it.